gazel.go

```diff
@@ -48,7 +50,42 @@ func main() {
 }
 
 type Venderor struct {
```

Comment:
shouldn't this be `Vendorer` rather than `Venderor`, given that the path is `vendor/`?

I guess the ship has already sailed.